lib/response_massager.js

```diff
@@ -8,7 +8,12 @@ class ResponseMassager {
   }
 
   massagedResponse (request, options) {
-    const updatedWithParallelFields = parallelFieldsExtractor(this.elasticSearchResponse)
+    const tempResponse = this.elasticSearchResponse
+    if (this.elasticSearchResponse.hits.hits[0].inner_hits && this.elasticSearchResponse.hits.hits[0].inner_hits.items) {
```

Comment:
I'm not sure this check is enough for every query. We should handle cases where there are zero matches for example (e.g. `/api/v0.1/discovery/resources?q=fladeedle`)